internal/audit/audit.go

```diff
@@ -71,3 +73,30 @@ type Record struct {
 	// Fields hold any additional context relevant to the Action
 	Fields []log.Field
 }
+
+type AuditLogSetting = int
+
+const (
+	GitserverAccess = iota
```

Comment:
This makes it the proper type:

```suggestion
	GitserverAccess AuditLogSetting  = iota
```

Reply:
I guess it doesn't matter? Learned something about iota again.

Reply:
Yeah, it doesn't :)